spec/src/main/asciidoc/batch_programming_model.adoc

```diff
@@ -1349,6 +1375,21 @@ parent step/thread; there is a distinct StepContext for each sub-thread
 and each StepContext has its own distinct persistent user data for each
 sub-thread.
 
+===== Additional requirements for CDI bean instances
+
+The batch runtime must ensure there is a producer for the batch context types with
+Dependent scope.
+
+The batch runtime should cache these Dependent scope instances to provide a view
+of the instance scope to the application aligning with the above section.
+
+(It is left undefined whether the application can override with its own producer,
```

Comment:
In theory, a user could always provide an Alternative producer to override it, so maybe we can remove this paragraph.

Reply:
Yeah, is there any value in saying "we thought of this case but didn't have anything to say" vs. just not saying anything.   Perhaps not.  

I know Romain had mentioned it in Slack so at least wanted to include it in the first round.

I'll remove it.

Reply:
REmoved